spec/support/features/upload_baskets.rb

```diff
@@ -0,0 +1,9 @@
+module Features
+  def upload_baskets(number)
+    number.times do
+      visit '/baskets/new'
+      page.attach_file('file', 'baskets/basket_2.txt')
```

Comment:
Here you're attaching the same basket multiple times, right? Or you need to interpolate on the file number?

Reply:
Yes, I'm attaching the same file multiple times. As I needed to test if the pagination link shows up when there are more than five receipts in the user's history.